lib/webmention/parsers/html_parser.rb

```diff
@@ -0,0 +1,73 @@
+module Webmention
+  module Parsers
+    class HtmlParser < BaseParser
+      def self.mime_types
+        ['text/html']
+      end
+
+      Parsers.register(self)
```

Comment:
When adding new Parsers, order here matters. First, define `self.mime_types` and return an array. Then call `Parsers.register(self)` and the Parser will register itself and be made available elsewhere.